### PR TITLE
Add/update Opera data for Window API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR updates the Opera (Presto) data for the Window API based upon manual testing to achieve more real data for the Window API. Some data is mirrored from Chrome.

Tests used: https://mdn-bcd-collector.appspot.com/tests/api/Window

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -2199,7 +2199,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "15"
```
Comment (foolip) [inline api/Window.json:2202]: This is post-Presto. There's a bunch of that, but the PR isn't too big, so I'll just change the description instead.

File: api/Window.json [hunk 2]
```diff
@@ -6243,10 +6246,10 @@
               "version_added": "9"
             },
             "opera": {
-              "version_added": true
+              "version_added": "3"
```
Comment (foolip) [inline api/Window.json:6249]: Is this based on testing Opera 3? It's the only so early version in this PR... If there's any doubt, then I'd prefer "≤12.1".

**Review by foolip — COMMENTED** [review]
Good stuff, just a few minor things to fix or back out.

File: api/Window.json (hunk 2 above)
Comment (vinyldarkscratch) [reply api/Window.json:6249]: I copied this from `pageYOffset`, and then confirmed the results!

**Review by foolip — APPROVED** [review]

**Pull request merged**